tests/functional/Tribe/Events/Importer/File_Importer_VenuesTest.php

```diff
@@ -64,70 +73,30 @@ public function it_should_be_instantiatable() {
 		$sut = $this->make_instance();
 	}
 
-	/**
-	 * @test
-	 * it should not mark record as invalid if featured image entry is missing
-	 */
-	public function it_should_not_mark_record_as_invalid_if_featured_image_entry_is_missing() {
-		$sut = $this->make_instance();
-
-		$post_id = $sut->import_next_row();
-
-		$this->assertNotFalse( $post_id );
-	}
-
-	/**
-	 * @test
-	 * it should import and attach featured image if featured image is ok
-	 */
-	public function it_should_import_and_attach_featured_image_if_featured_image_is_ok() {
-		$image_url     = plugins_url( '_data/csv-import-test-files/featured-image/images/featured-image.jpg', codecept_data_dir() );
-		$attachment_id = $this->factory()->attachment->create_upload_object( $image_url );
-		$this->featured_image_uploader->upload_and_get_attachment()->willReturn( $attachment_id );
 
-		$sut = $this->make_instance();
+	protected function make_instance( $template_dir = null ) {
+		$this->setup_file( $template_dir );
 
-		$post_id = $sut->import_next_row();
+		$sut = new Venues_Importer( $this->file_reader, $this->featured_image_uploader->reveal() );
```

Comment:
It's common to use a var name like that to have a clear representation that:

* you are actually testing **one** thing only
* what you are testing